test.css

```diff
@@ -0,0 +1,54 @@
+@media (max-width: 765px) {
+  .menu {
+    display: block;
+    cursor: pointer;
+  }
+
+  .specific-links {
+    display: none;
+  }
+
+  .menu-popup {
+    position: absolute;
+    z-index: 9999;
+    top: 0;
+    left: 0;
+    background-color: #3c3a39;
+    backdrop-filter: blur(6px);
+    width: 100%;
+    height: 100%;
+    display: flex;
+  }
```

Comment:
- [ ] Please make sure to add the required styles that match the original design. Please look at the screenshot below of your website and compare it with the original Figma design and apply the necessary changes for the colors, font weight, size, and spacing (margin and padding).
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/225313188-d74989e3-694e-493e-b167-1da46f70d60e.png)